shaders/palette.frag

```diff
@@ -65,13 +66,15 @@ void main() {
 
     vec4 remap = texture(remaps, vec2(texel.r, remap_offset / 18.0));
 
+    bool NO_REMAP = USE_HAR_QUIRKS && index > 0x30;
```

Comment:
Wont this only work for HAR 1 ?

Reply:
No, the `index` is taken directly from the input texture-- which is the same for both players.

Reply:
a previous version of the patch used the texel.r (after pal_offset had been applied), but I thought using the unmodified texture data would lend itself better to out of order execution.

Reply:
Har 1 palette is 1-48, and HAR2 palette is the next 48 colors. This will disable remaps for entire har2 palette when USE_HAR_QUIRKS is enabled for it, yes ?

Reply:
Oh wait, never mind, I misread the file. This checks against the sprite index.